indico/web/templates/layout/dialog_base.html

```diff
@@ -0,0 +1,24 @@
+{% block page %}
+    <div class="{% block dialog_class %}dialog-page{% endblock %}"
+        {% if self.dialog_id()|trim -%}
```

Comment:
I think both solutions look awful, so I have no personal preference. Anyway, I think this is the kind of thing that can be left to personal preference 😉